systemtests/scripts/test_func.sh

```diff
@@ -29,7 +29,8 @@ function setup_test() {
     oc login -u ${OPENSHIFT_USER} -p ${OPENSHIFT_PASSWD} --insecure-skip-tls-verify=true ${OPENSHIFT_URL}
     oc adm --config ${KUBEADM} policy add-cluster-role-to-user cluster-admin $OPENSHIFT_USER
     export OPENSHIFT_TOKEN=`oc whoami -t`
-    ansible-playbook ${ENMASSE_DIR}/ansible/playbooks/openshift/systemtests.yml --extra-vars "namespace=${OPENSHIFT_PROJECT} admin_user=${OPENSHIFT_USER}"
+    ansible-playbook ${CURDIR}/../ansible/playbooks/systemtests-dependencies.yml
+    ansible-playbook ${ENMASSE_DIR}/ansible/playbooks/openshift/install.yml -i ${CURDIR}/../ansible/inventory/systemtests.inventory --extra-vars "namespace=${OPENSHIFT_PROJECT} admin_user=${OPENSHIFT_USER}"
```

Comment:
Maybe I'm just confused, but I think ENMASSE_DIR will point to 'systemtests' in this case, and not 'templates/install', so the install playbook does not exist there.

Reply:
@lulf I will rename ENMASSE_DIR you're right name is confusing, but install playbook is still there it is your install playbook but I run this playbook with systemtests.inventory